src/material/sidenav/drawer.ts

```diff
@@ -403,23 +405,47 @@ export class MatDrawer implements AfterContentInit, AfterContentChecked, OnDestr
     return this.toggle(false);
   }
 
+  /** Closes the drawer with context that the backdrop was clicked. */
+  _closeViaBackdropClick(): Promise<MatDrawerToggleResult> {
+    // If the drawer is closed upon a backdrop click, we always want to restore focus. We
+    // don't need to check whether focus is currently in the drawer, as clicking on the
+    // backdrop causes blurring of the active element.
```

Comment:
FWIW it's not actually the click that causes the blurring, it gets moved on `mousedown`.

Reply:
Sure, but in this case a simple mousedown doesn't lead to `_closeViaBackdropClick` so in the context of a click, the active element is blurred and other situations are not relevant.